test/httpServer.test.js

```diff
@@ -320,23 +353,51 @@ describe("httpServer", { concurrency: 1 }, () => {
   });
 });
 
+// describe("httpServer without options", () => {
+//   let httpServer;
+//   let opened = false;
+//   // We want to disable WS
+//   process.env.NODE_ENV = "test";
+
+//   before(async() => {
+//     const module = await esmock("../src/http-server/index.js", {
+//       open: () => (opened = true)
+//     });
+
+//     httpServer = module.buildServer(JSON_PATH);
+//     await once(httpServer.server, "listening");
+//     enableDestroy(httpServer.server);
+//   });
+
+//   after(async() => {
+//     httpServer.server.destroy();
+//   });
+
+//   test("should listen on random port", () => {
+//     assert.ok(httpServer.server.address().port > 0);
+//   });
+
+//   test("should have openLink to true", () => {
+//     assert.equal(opened, true);
+//   });
+// });
 describe("httpServer without options", () => {
   let httpServer;
   let opened = false;
-  // We want to disable WS
   process.env.NODE_ENV = "test";
 
-  before(async() => {
-    const module = await esmock("../src/http-server/index.js", {
-      open: () => (opened = true)
+  before(async () => {
+    const openMock = mock.method(indexModule, "open", () => {
+      opened = true;
     });
 
-    httpServer = module.buildServer(JSON_PATH);
+    httpServer = indexModule.buildServer(JSON_PATH);
     await once(httpServer.server, "listening");
     enableDestroy(httpServer.server);
+    openMock.mock.restoreAll();
   });
 
-  after(async() => {
+  after(async () => {
```

Comment:
```suggestion
  after(async() => {
```
Make sure you have the linter configured in your IDE, you can also run `npm run lint-fix` (there are some lint errors in the code)

Reply:
fixed all async spacing typos in latest commit